### PR TITLE
Experimental Cassandra Support

### PR DESCRIPTION
**Pull request by beberlei**
Using the [new php driver](https://github.com/datastax/php-driver) to add Cassandra Support.

It works with the K/V part already, only the range support is now missing.

Closes https://github.com/doctrine/KeyValueStore/issues/38

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -5,12 +5,12 @@
     },
     "require-dev": {
         "riak/riak-client": "dev-master",
-        "microsoft/windowsazure": "dev-master"
-    },          
+        "datastax/php-driver": "dev-master#53846f096a2a8d81bc3300fe8da64911838b34b8"
```
Comment (stof) [inline composer.json]: what is the goal of this requirement ?

**Comment by EmanueleMinotto** [comment]
ping @Ocramius

File: .travis.yml [hunk 2]
```diff
@@ -9,6 +7,14 @@ php:
   - hhvm
 
 before_install:
+  - if [[ $TRAVIS_PHP_VERSION != "hhvm" && $TRAVIS_PHP_VERSION != "7.0" ]]; then sudo sh -c "echo 'JVM_OPTS=\"\${JVM_OPTS} -Djava.net.preferIPv4Stack=false\"' >> /usr/local/cassandra/conf/cassandra-env.sh"; fi
```
Comment (Ocramius) [inline .travis.yml]: All this stuff should be moved to a bash script

File: lib/Doctrine/KeyValueStore/Storage/CassandraStorage.php [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+            $keys[] = $name;
+            $values[] = $value;
+        }
+
+        foreach ($data as $name => $value) {
+            $keys[] = $name;
+            $values[] = $value;
+        }
+
+        $cql = "INSERT INTO " . $storageName . " (" . implode(', ', $keys) . ") VALUES (" . implode(', ', array_fill(0, count($values), '?')) . ")";
```
Comment (Ocramius) [inline lib/Doctrine/KeyValueStore/Storage/CassandraStorage.php]: Do you need an assignment here?

**Comment by Ocramius** [comment]
@EmanueleMinotto looks good overall, just that `.travis.yml` needs some cleanup ;-)

**Comment by EmanueleMinotto** [comment]
@Ocramius moved scripts into `tests/travis.sh`, is it ok or too much generic (it contains only Cassandra setup)?

**Comment by zeroedin-bill** [comment]
:+1:

**Comment by Ocramius** [comment]
@EmanueleMinotto :+1:

**Pull request merged**